recipe-server/client/control/state/action-types.js

```diff
@@ -0,0 +1,38 @@
+/* Keep this list alphabetized */
+
+export const ACTION_FETCH = 'ACTION_FETCH';
```

Comment:
I don't think we should use `module.exports`. It makes more typing, but it makes the code easier to understand, both for humans and for linters. For example, this proposal will break the import linting we have that detects if we typo a module import.